python/lsst/ip/diffim/getTemplate.py

```diff
@@ -182,6 +182,119 @@ def run(self, exposure, sensorRef, templateIdList=None):
         return pipeBase.Struct(exposure=coaddExposure,
                                sources=None)
 
+    def assembleTemplateExposure(self, butlerQC, skyMapRef, coaddExposureRefs, exposure):
```

Comment:
Also, once the Gen 2 butler is gone, won't this be the `run` method? Maybe this should be called `runGen3` to anticipate that change. Or, the current `run` method could be renamed to `runGen2` and this could become `run`.